### DISCUSSION TITLE
Rookies Takımı Başvuru

### DISCUSSION THREAD

**Discussion by CinarSatir** [sonsuzus/BilsemPython#61] [Genel]
Rookies Takımına katılmak istiyorsanız yorumlara isminizi ve oyg gurubunuzu atabilirsiniz. 
Verimli çalışabilecek , takımımızı yükseltebilecek ve kendine güvenen ekip arkadaşları arıyorum. Takım seçmede bol şans :+1:

**Comment by OmerMete** [comment]
Ben başvurmak istiyorum adım Ömer Mete Şakar ÖYG 2 perşembe günleri 3 ve 4. derslere giriyorum 

(Yanlış anlamadıysam böyle başvuru oluyor)

**Reply by CinarSatir** [reply]
Takıma hoşgeldin Ömer Mete whatsap tan 05462462705 numarasına adını ve soyadını at gurup oluşturacağım.

**Comment by Efemert2010** [comment]
ben katılmak istiyorum adım efe mert cuma ÖYG 1 Çarşamba günü 1. ve 2. derslere giriyorum

**Reply by CinarSatir** [reply]
Takıma hoşgeldin Efe Mert whatsap tan 05462462705 numarasına adını ve soyadını at guruba ekleyeyim seni.

**Comment by OmerB2** [comment]
Ben başvurmak istiyorum adım Ömer Bayındır ÖYG 1 Çarşamba günü 1. ve 2. derslere giriyorum

**Reply by CinarSatir** [reply]
Takıma hoşgeldin Ömer whatsap tan 05462462705 numarasına adını ve soyadını at gurup oluşturacağım.

**Comment by QLNH9491** [comment]
Ben de başvurmak istiyorum adım Ömerasaf ÇİÇEK ÖYG 1 Çarşamba günü 3. ve 4. derslere giriyorum.

**Comment by Sadik1Dev** [comment]
Merhaba, ben Sadık bu gruba katılmak istiyorum, cumartesi günü ÖYG1 3. ve 4. derslere giriyorum.

**Reply by CinarSatir** [reply]
Takıma hoşgeldin Sadık whatsap tan adını ve soyadını 05462462705 numarasına atarsan seni guruba ekleyebilirim.

**Reply by CinarSatir** [reply]
Sadık perşembe akşamına kadar bilgilerini whatsap tan atarsan sevinirim

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Comment by CinarDumanli12** [comment]
Merhaba ben Çınar katılmak istiyorum ÖYG-1 Çarşamba 16.30-18.00

**Comment by Mertcakm** [comment]
Merhaba ben Mert ÖYG-1 çarşamba 18:00-19:30 grubundayım ve aranıza katılmak istiyorum.

**Comment by Muhammedipek** [comment]
Ben Muhammed İpek öyg 2 öğrencisiyim perşembe 3 4 derslerine giriyorum Ömer Mete arkadaşım takıma yararlı olacağımı düşünüyorum
